doc/design.rst

```diff
@@ -606,3 +597,61 @@ are deleted, the particular snapshot vanished and all snapshots
 depending on data that has been added in the snapshot cannot be restored
 completely. Restic is not designed to detect this attack.
 
+With the aforementioned assumptions and guarantees in mind, the following  are
+examples of things an adversary could achieve in various circumstances.
+
+An adversary with access to your backup storage location could: 
+- Delete or manipulate your backups, thereby impairing your ability to restore
+files from the compromised storage location. - Attempt a brute force password 
+guessing attack against a copy of the  repository (even more reason to use 
+long, random passwords).
+
+An adversary with write access to your files at the storage location could: 
```

Comment:
"Write access" seems to primarily mean "delete files" here. Or is this intended to also include an attacker which just can add (broken) data?

Reply:
Both. That's what the "or manipulate your backups" after the delete intended to convey.